docs/docs/aws.md

```diff
@@ -717,13 +724,21 @@ install_dependencies () {
 install_dependencies $LIB_PATH $ICEBERG_MAVEN_URL $ICEBERG_VERSION "${ICEBERG_PACKAGES[@]}"
 ```
 
+For versions after 7.1.0 there is an specific config that can be used to enable data prefecth optimization. You just need to add the following property on your Spark config.
+
+```shell
+spark.sql.iceberg.data-prefetch.enabled=true
```

Comment:
I don't believe this is an Iceberg property.  If this is specific to EMR, I don't believe it should be included here.

Reply:
this is specific to EMR yes ( internal iceberg runtime), however we are on the "aws" docs page.

 I think that stating that you can add that parameter to improve the performance of AWS is good to have/know?